test/test_kubeclient.rb

```diff
@@ -459,107 +428,100 @@ def test_api_basic_auth_failure
     stub_request(:get, 'http://username:password@localhost:8080/api/v1')
       .to_raise(KubeException.new(401, error_message, response))
 
-    client = Kubeclient::Client.new('http://localhost:8080/api/',
-                                    auth_options: {
-                                      username: 'username',
-                                      password: 'password'
-                                    })
+    client = Kubeclient::Client.new(
+      'http://localhost:8080/api/',
+      auth_options: { username: 'username', password: 'password' }
+    )
 
     exception = assert_raises(KubeException) { client.get_pods }
     assert_equal(401, exception.error_code)
     assert_equal(error_message, exception.message)
     assert_equal(response, exception.response)
-    assert_requested(:get,
-                     'http://username:password@localhost:8080/api/v1',
-                     times: 1)
+    assert_requested(:get, 'http://username:password@localhost:8080/api/v1', times: 1)
   end
 
   def test_init_username_no_password
     expected_msg = 'Basic auth requires both username & password'
     exception = assert_raises(ArgumentError) do
-      Kubeclient::Client.new('http://localhost:8080',
-                             auth_options: {
-                               username: 'username'
-                             })
+      Kubeclient::Client.new(
+        'http://localhost:8080',
+        auth_options: { username: 'username' }
+      )
     end
     assert_equal(expected_msg, exception.message)
   end
 
   def test_init_user_no_password
     expected_msg = 'Basic auth requires both username & password'
     exception = assert_raises(ArgumentError) do
-      Kubeclient::Client.new('http://localhost:8080',
-                             auth_options: {
-                               user: 'username'
-                             })
+      Kubeclient::Client.new(
+        'http://localhost:8080',
+        auth_options: { user: 'username' }
+      )
     end
     assert_equal(expected_msg, exception.message)
   end
 
   def test_init_username_and_bearer_token
     expected_msg = 'Invalid auth options: specify only one of username/password,' \
-                   ' bearer_token or bearer_token_file'
+      ' bearer_token or bearer_token_file'
```

Comment:
consider dropping the start of the string to a new line (as you did elsewhere), to keep alignment.